internal/api/webhook.go

```diff
@@ -43,8 +43,8 @@ func handleCreateWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.WriteHeader(http.StatusOK)
 	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
```

Comment:
```suggestion
	w.WriteHeader(http.StatusAccepted)
```

Reply:
ahh my bad, overlooked
thanks for catching that!